pkg/controller/buildrun/generate_taskrun_test.go

```diff
@@ -19,132 +19,45 @@ import (
 var _ = Describe("GenerateTaskrun", func() {
 
 	var (
-		build                                *buildv1alpha1.Build
-		buildRun                             *buildv1alpha1.BuildRun
-		buildStrategy                        *buildv1alpha1.BuildStrategy
-		builderImage                         *buildv1alpha1.Image
-		clusterBuildStrategy                 buildv1alpha1.BuildStrategyKind
-		dockerfile, buildah, buildpacks, url string
+		build                       *buildv1alpha1.Build
+		buildRun                    *buildv1alpha1.BuildRun
+		buildStrategy               *buildv1alpha1.BuildStrategy
+		builderImage                *buildv1alpha1.Image
+		dockerfile, buildpacks, url string
+		t                           test.Catalog
```

Comment:
Just my subjective opinion: I would have expected `catalog` or `cat`. `t` in the context of Golang unit test reminds me of `testing.T`. Note: I did not notice this upfront. I read through your code and found `t` there and it was not clear to me where it was going to. So, I scrolled up. I can only guess that for another variable name, I would not have needed to do this.

Reply:
is a good point. I will not do this now because I want to keep this particular `generate_taskrun_test.go ` with fewest changes as possible. But in general you re on the right side. Feel free to PR later this change.

Reply:
I modified this as requested, but I use `ctl` as we used in all the other unit tests. Resolving